build.sc

```diff
@@ -119,6 +131,7 @@ trait CommonJsModule extends CommonPublishModule with ScalaJSModule{
   def platformSegment = "js"
   def crossScalaJSVersion: String
   def scalaJSVersion = crossScalaJSVersion
+  def scalacOptions = super.scalacOptions()
```

Comment:
I don't know why, but if I remove this line some modules loose the `-scalajs` option.
Couldn't find where this happens since there is no redefinition of `scalacOptions` without `super.scalacOptions() ++`